app/serializers/submission_serializer/definitions/address.rb

```diff
@@ -0,0 +1,15 @@
+module SubmissionSerializer
+  module Definitions
+    class Address < Definitions::BaseDefinition
+      def to_builder
+        Jbuilder.new do |json|
```

Comment:
We need to also add the `lookup_id` attribute as it will be needed (eventually) for re-hydration.